doc/sphinx/source/changelog.rst

```diff
@@ -3,6 +3,119 @@
 Changelog
 =========
 
+.. _changelog-v2-11-0:
+
+v2.11.0
+-------
+Highlights
+
+- Two new recipes have been added: aerosol optical depth climatology and CMIP6
```

Comment:
It would be nice if there was a short description here and a link to the documentation page, e.g. 
"Recipe [recipe_aod_aeronet_assess.yml](https://docs.esmvaltool.org/en/latest/recipes/recipe_aod_aeronet_assess.html) evaluates model aerosol optical depth (AOD) climatologies against ground based observations from the AeroNET measurement network."
and something similar for the other recipe

Reply:
Done in e88739363 (with small typo fix in 7ba8da5b7) 😊